Realm/Tests/MigrationTests.mm

```diff
@@ -200,6 +200,22 @@ - (void)assertNoMigrationRequiredForChangeFrom:(NSArray *)from to:(NSArray *)to
     RLMAssertRealmSchemaMatchesTable(self, [RLMRealm realmWithConfiguration:config error:nil]);
 }
 
+- (void)assertDeletingRealmFilesForChangeFrom:(NSArray *)from to:(NSArray *)to {
```

Comment:
since this is just used from one caller (`testDeleteRealmIfMigrationNeeded`) it would seem preferable to keep the logic there rather than have this helper method?